src/ethereum_spec_tools/evm_tools/daemon.py

```diff
@@ -57,6 +57,8 @@ def do_POST(self) -> None:
             f"--state.chainid={content['state']['chainid']}",
             f"--state.reward={content['state']['reward']}",
         ]
+        if 'state_test' in content and content['state_test']:
+            args.append("--state-test")
```

Comment:
My concern here is the same as in #1006: detecting flags based on the content of the JSON makes the command line interface inconsistent. If the t8n CLI requires `--state-test` but t8n as a daemon does not, that feels weird.

Originally this file parsed the command line arguments from the query string. Should we bring that back?

https://github.com/ethereum/execution-specs/blob/457529110356df26284c38b449b672a0f2e802d8/src/ethereum_spec_tools/evm_tools/daemon.py#L36-L37

Reply:
I understand, and it makes sense that we need to standardize this, and also if that makes the code on your end cleaner it would be ok for us too.